Docker.DotNet/ContainerOperations.cs

```diff
@@ -134,6 +134,23 @@ public Task<Stream> ExportContainerAsync(string id, CancellationToken cancellati
             return response.StatusCode != HttpStatusCode.NotModified;
         }
 
+        public async Task<ExecContainerResponse> ExecContainerAsync(string id, Config config)
```

Comment:
instead of directly passing in config, please create a `ExecContainerParameters` and have Config as a field in there (see rest of the file for examples)